app/views/layouts/application.html.erb

```diff
@@ -128,6 +128,18 @@
       <%= flash_messages %><%# These are from the controller via the `flash` array %>
       <%= yield(:flashes) %><%# These are from the view via `content_for` %>
 
+      <% if show_back_link(controller) %>
+        <div class="visible-xs mobile-back">
+          <h5>
+            <% index_link = index_path(controller.controller_name) %>
+            <%= link_to(index_link[:url]) do %>
+              <i class="fa fas fa-angle-left"></i>
+              Back to <%= index_link[:label] %>
+            <% end %>
+          </h5>
+        </div>
+      <% end %>
```

Comment:
The links look really cool! I love that you've thought about this in a systematic way instead of a one off.

I think we should adjust the pattern a bit though, to match the way things are done in Gather.

Can we have a `yield` here, similar to how `yield(:action_links)` works below? And then on the various show pages, you'd define a block with the actual `link_to` call in it? For pages that don't need a back link, you'd just not define the block.